perllib/FixMyStreet/App/Controller/Around.pm

```diff
@@ -38,6 +38,11 @@ sub index : Path : Args(0) {
     # Check if we have a partial report
     my $partial_report = $c->forward('load_partial');
 
+    # Check if the user is searching for a report by ID
+    if ( $c->get_param('pc') =~ /^\s*ref:(\d+)\s*$/ ) {
```

Comment:
Could this also, for a UK cobrand, allow searching without `ref:` (as we don't have numeric zipcodes)?